routing/turns_tts_text.hpp

```diff
@@ -39,10 +41,19 @@ class GetTtsText
 
 private:
   std::string GetTextById(std::string const & textId) const;
+  std::string GetTextByIdTrimmed(std::string const & textId) const;
 
   std::unique_ptr<platform::GetTextById> m_getCurLang;
-};
 
+  /// \brief Removes a terminal period (or CJK or Hindi equivalent) from a string
+  /// @param s - String to be modified
+  static void RemoveLastDot(std::string & s)
```

Comment:
Is the goal to remove the last unicode dot from the string? Then it's not the case. Functions below remove any last point in the sentence, including "test.string" => "teststring".

Use base::EatSuffix(".") instead.